test/integration/assorted.js

```diff
@@ -10,11 +10,22 @@ const {execCli} = require('../helper/cli');
 test('timeout', t => {
 	execCli(['long-running.js', '-T', '1s'], (err, stdout) => {
 		t.ok(err);
-		t.match(stdout, /Exited because no new tests completed within the last 1000ms of inactivity/);
+		t.match(stdout, /Timed out/);
 		t.end();
 	});
 });
 
+test('interrupt', t => {
+	const proc = execCli(['long-running.js', '-T', '5s'], (err, stdout) => {
```

Comment:
Perhaps remove the `-T` argument to make it clearer this only deals with `SIGINT`.

Reply:
I've removed this test altogether since it didn't work during `npm test`. For reference, here it was:

```
test('interrupt', t => {
	const proc = execCli(['long-running.js'], (_, stdout) => {
		t.match(stdout, /SIGINT/);
		t.end();
	});

	setTimeout(() => {
		proc.kill('SIGINT');
	}, 2000);
});
```